packages/cli/src/lib/cmds/doctor.ts

```diff
@@ -19,9 +19,10 @@ import {join} from 'path';
 import {existsSync, promises as fsPromises} from 'fs';
 import {loadOrCreateConfig} from '../config';
 import {enUS as messages} from '../strings';
+import {getJavaHome} from '../../../../core/src/lib/jdk/JdkHelper';
 
 async function jdkDoctor(config: Config, log: Log): Promise<boolean> {
-  const jdkPath = config.jdkPath;
+  const jdkPath = getJavaHome(config, process);
```

Comment:
What about moving the entire path validation to a static function in `JdkHelper` and just using it in doctor?

Reply:
We could move the androidSdk check to AndroidSdkTools, and remove code duplication here: https://github.com/GoogleChromeLabs/bubblewrap/blob/master/packages/core/src/lib/androidSdk/AndroidSdkTools.ts#L44-L46